_episodes_rmd/13-dplyr.Rmd

```diff
@@ -181,6 +181,17 @@ even better.
 > >lifeExp_bycountry <- gapminder %>%
 > >    group_by(country) %>%
 > >    summarize(mean_lifeExp=mean(lifeExp))
+> >
+> >lifeExp_bycountry$country[which.max(lifeExp_bycountry$mean_lifeExp)]
+> >lifeExp_bycountry$country[which.min(lifeExp_bycountry$mean_lifeExp)]
+> >```
+> Another way to do this is to use the `dplyr` function `arrange()`, which has 
```

Comment:
Also I might also might modify the challenge itself to clarify what is being asked for - is it the min/max individual value or the lowest/highest mean value that we'd like to find?